### PR TITLE
feat(extension) add onboarding events to PostHog

### PR DESCRIPTION
**Pull request by DominikGuzei**
# Checklist

- [x] [JIRA](https://input-output.atlassian.net/browse/LW-6768)

---

## Proposed solution

This PR adds tracking for the following PostHog events:

- wallet | change active page | view
- onboarding | new wallet | all done | go to my wallet | click
- onboarding | hardware wallet | all done | go to my wallet | click
- onboarding | restore wallet | all done | go to my wallet | click

## Testing

You need access to our PostHog service and go through the steps required to trigger those events + check them in PostHog

### CODE REVIEW

File: apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/components/WalletSetup.tsx [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 import { ILocalStorage } from '@src/types';
 import { deleteFromLocalStorage, getValueFromLocalStorage } from '@src/utils/local-storage';
+import { WalletSetupLayout } from '@src/views/browser-view/components/Layout';
+import { WarningModal } from '@src/views/browser-view/components/WarningModal';
+import React, { useCallback, useEffect, useState } from 'react';
+import { Route, Switch, useHistory, useRouteMatch } from 'react-router-dom';
+import { HardwareWalletFlow } from './HardwareWalletFlow';
 import { Portal } from './Portal';
+import styles from './WalletSetup.module.scss';
+import { WalletSetupWizard } from './WalletSetupWizard';
```
Comment (DominikGuzei) [inline apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/components/WalletSetup.tsx:19]: Sorry for these unrelated import code formatting changes, seems like this file hasn't been run through linters/formatters for a while 😉

**Review by marcin-mazurek — DISMISSED** [review]

**Review by jplorek-atix — APPROVED** [review]
LGTM

**Comment by bslabiak** [comment]
Not able to generate `onboarding | hardware wallet | all done | go to my wallet | click` event on ledger and trezor device.

File: apps/browser-extension-wallet/src/providers/AnalyticsProvider/context.tsx [hunk 2]
```diff
@@ -50,5 +50,14 @@ export const AnalyticsProvider = ({
     analyticsTracker.setSiteId(currentChain);
   }, [currentChain, analyticsTracker]);
 
+  // Track page changes with PostHog in order to keep the user session alive
+  useEffect(() => {
+    const trackActivePageChange = () => analyticsTracker.sendEventToPostHog(PostHogAction.WalletChangeActivePage);
```
[the remark this answers is not in the document]
Comment (DominikGuzei) [reply apps/browser-extension-wallet/src/providers/AnalyticsProvider/context.tsx]: Done ✅

**Comment by DominikGuzei** [comment]
@bslabiak I could reproduce the issue regarding hardware wallets and pushed a fix to ensure that they are always tracked. 

What was the issue? There was a race condition due to a workaround for Ledger that the page is reloaded and this sometimes prevented sending the analytics event

**Comment by DominikGuzei** [comment]
@bslabiak also the `event: wallet | change active page | view` should be correctly tracked as pageview now

**Review by bslabiak — APPROVED** [review]
